pkgs/games/pegasus-frontend/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, cmake
+, libsForQt5
+, wrapQtAppsHook
+, SDL2
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pegasus-frontend";
+  version = "0-unstable-2023-05-22";
```

Comment:
```suggestion
  version = "unstable-2023-05-22";
```

Reply:
The current packaging guidelines require last version number before the `unstable` part: https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
Just searched nixpkgs, this could be the first `0-unstable` version of a package. The `pegasus-frontend` repository uses `alpha16` or weekly releases to denote version. I'm fine whichever.

Reply:
Yeah, the policy changed pretty recently.

Reply:
Those likely predate the policy. It has only been changed two weeks ago: https://github.com/NixOS/nixpkgs/pull/234201

Reply:
bump @SuperSandro2000

Reply:
Hello, sorry for bumping this one again @SuperSandro2000 I hope my changes is in order.

Reply:
We are not going to bother with this now because if you followed the RFC you can ask 5 people and get 8 answers how this is supposed to be done.